src/index.js

```diff
@@ -83,6 +84,9 @@ class LndEngine {
       this[configKey] = this.currencyConfig[configKey]
     })
 
+    this.outboundTimeLock = outboundTimeLock || 9 * this.secondsPerBlock
```

Comment:
sorry, this is a [nit]. Would be worth to have some kind of documentation to show where `9` and `144` came from